openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1500,9 +1500,28 @@ Credential Refresh can be initiated by the Wallet independently from the Credent
 
 It is up to the Credential Issuer whether to update both the signature and the claim values, or only the signature.
 
-# Privacy Considerations
+# Privacy Considerations {#privacy-considerations}
 
-TBD
+Privacy harms can occur if information about a person is released to another party without the person's consent.
+True consent involves both the person knowing what information is being released and knowing who it is being released to, and for what purpose.
+The means to ensure the session's integrity to prevent disclosure to unintended parties, including wallets not under control of the End-User, are means to protect privacy.
+
+The Issuer needs to make choices about which claims are selectively disclosable (and should do so carefully).
+Claims that are always disclosed can enable correlation by verifiers.
+
+Multiple credentials can be linkable.
+Batch issuance solves this potential privacy issue, but only when used correctly.
```

Comment:
Gabe: `but only when used correctly` is not clear. -> `Batch Issuance can solve~`.